manifests/shared-workarounds.yaml

```
@@ -67,3 +67,14 @@ postprocess:
     }
     EOF
     chmod +x /usr/lib/dracut/modules.d/36coreos-multipath-fix/module-setup.sh
+
+  # Fedora 37 adds the nvmf dracut module to the initrd, causing
+  # ext.config.files.dracut-executable to notice that the module puts a
+  # non-executable file in /usr/sbin.  Dracut has been updated to fix the
+  # missing permission, and hopefully this can be removed for Fedora 38.
+  # https://github.com/coreos/fedora-coreos-tracker/issues/1155
+  # https://github.com/dracutdevs/dracut/pull/1777
+  - |
+    #!/usr/bin/env bash
+    set -xeuo pipefail
+    chmod +x /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
```

WDYT about having it targetted at just F37? AFAIU this isn't a problem in F35/F36 and we hope it will be fixed in F38 (if not we'd still like to know I think).

```suggestion
    #!/usr/bin/env bash
    set -xeuo pipefail
    source /etc/os-release
    if [ ${VERSION_ID} -eq 37 ]; then
        chmod +x /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
    fi
```

Also - this file is "shared-workarounds.yaml". I think we put stuff in here that needs to apply to both FCOS and RHCOS. Hopefully this one never hits RHCOS (we caught it early enough). I'm fine with it staying here, but just saying we might want to consider putting it somewhere else.

---

It's not causing failures on F35/F36/RHCOS, but the bug is latent there too, so I figured it made sense to just fix it everywhere.  Removing it in the F38 timeframe could have consequences for RHCOS, though, so I'll move it out of `shared-workarounds`.

---

ahh. I see. The bug exists in those other places, it's just that we don't pull in that module yet in those other streams so the test won't fail.